src/Parser/WindwalkerMonk/Modules/Spells/SEF.js

```diff
@@ -0,0 +1,40 @@
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SPELLS from 'common/SPELLS';
+
+class SEF extends Module{
```

Comment:
Please name this class (and its file) the full name (StormEarthAndFire) so it is much easier to spot in a long list of names.